package proto_ff;

import "yd_fieldoptions.proto";


message godHoodgodHoodattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 up = 2[(yd_fieldoptions.field_cname) = "增量"];
	optional int32 value = 3[(yd_fieldoptions.field_cname) = "值"];
}

message godHoodgodHood
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "灵魄ID"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 attrNum = 3[(yd_fieldoptions.field_cname) = "属性数"];
	optional int32 quality = 4[(yd_fieldoptions.field_cname) = "品质"];
	optional int32 lvUp = 5[(yd_fieldoptions.field_cname) = "等级上限"];
	optional int32 basicsExp = 6[(yd_fieldoptions.field_cname) = "分解的基础经验值"];
	repeated godHoodgodHoodattributeDesc attribute = 7[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_godHoodgodHood
{
	repeated godHoodgodHood godHoodgodHood_List = 1[(yd_fieldoptions.field_arysize)=200];
}

message godHoodexp
{
	optional int32 lv = 1[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 exp = 2[(yd_fieldoptions.field_cname) = "升级经验值"];
}


message Sheet_godHoodexp
{
	repeated godHoodexp godHoodexp_List = 1[(yd_fieldoptions.field_arysize)=300];
}
